linera-views/src/batch.rs

```diff
@@ -186,7 +186,7 @@ impl Batch {
     }
 
     /// The total size of the batch
-    pub fn size(&self) -> usize {
+    pub fn len(&self) -> usize {
```

Comment:
I understand why you'd like to imitate `Vec::len` but we're already using "batch size" everywhere and I believe it's totally fine.

Reply:
Ok, so let's revert that.